jp2_online/templates/captura/familia_form.html

```diff
@@ -24,8 +24,9 @@
             <div class="ln_solid"></div>
               <div class="form-group">
                 <div class="col-md-6 col-sm-6 col-xs-12 col-md-offset-3">
-                  <button class="btn btn-primary" type="reset">Reset</button>
-                  <button type="submit" class="btn btn-success">Guardar</button>
+                  <button id="submit_familia" type="submit" class="btn btn-danger">
```

Comment:
Also this should have the arrow to the right and say "Integrantes" instead of siguiente (to keep consistency w/everything else)